_delphi_utils_python/delphi_utils/logger.py

```diff
@@ -0,0 +1,58 @@
+"""Structured logger utility for creating JSON logs in Delphi pipelines."""
+import logging
+import sys
+import structlog
+
+def get_structured_logger(name=__name__):
+    """Create a new structlog logger.
+    
+    Use the logger returned from this in indicator code using the standard 
+    wrapper calls, e.g.:
+
+    logger.warning("Error", type="Signal too low").
```

Comment:
can we add a note on what the `name` arg is for, and also a line showing the initialization in this example?

Reply:
Done.